packages/@aws-cdk/aws-lambda-python-alpha/README.md

```diff
@@ -124,6 +124,20 @@ new python.PythonFunction(this, 'function', {
 });
 ```
 
+**Including hashes and excluding URLs**
+
+You can include hashes and exclude URLs in `poetry` as follows:
+
+```ts
+new python.PythonFunction(this, 'function', {
+  entry: '/path/to/poetry-function',
+  runtime: Runtime.PYTHON_3_8,
+  bundling: {
+    poetryIncludeHashes: true,
+    poetryWithoutUrls: true,
+  },
+});
+```
 
```

Comment:
```suggestion

**Excluding URLs*

You can exclude URLs in `poetry` using the optional boolean parameter `poetryWithoutUrls`:

```ts
new python.PythonFunction(this, 'function', {
  entry: '/path/to/poetry-function',
  runtime: Runtime.PYTHON_3_8,
  bundling: {
    poetryWithoutUrls: true,
  },
});
```
```